src/main/java/org/genomenexus/vep_wrapper/LinesToJSONListFilterOutputStream.java

```diff
@@ -0,0 +1,96 @@
+package org.genomenexus.vep_wrapper;
+
+import java.io.*;
+
+/** Accepts lines of valid JSON elements (one per line) and outputs a valid JSON list
+*/
+public class LinesToJSONListFilterOutputStream extends FilterOutputStream {
+
+    private boolean someDataHasBeenWritten = false;
+    private boolean lastWriteEndedLineWithoutBeginningAnother = true;
+
+    /** Creates an output stream filter built on top of the specified underlying output stream
+    */
+    public LinesToJSONListFilterOutputStream(OutputStream out) {
+        super(out);
+    }
+ 
+    /** Outputs the ending JSON list bracket
+    */
+    public void	complete() throws IOException {
```

Comment:
looks like a tab was inserted here before the method name?

Reply:
fixed